Updated with new content.

jiglei opened the pull request:
Items added:

- Milletian Weapon
- Milletian Armor
- The Boof of the Milletian
- Milletian Focus and Shield
- Red Kitty Necklace (Why not lol)
-105 Earrings (future equipment)

I added extra equipment type to make sure the milletian focus and shield can use the correct scrolls (Truth and Sealed)

Scrolls added:
Wish, Meaningful, Cold-blooded(Future ES)

jiglei committed:
Add files via upload

JamEnergy commented on data.js:
```diff
@@ -26,15 +26,16 @@ var g_charactersLookup = {
 		"smallshield":true,
-		"largeshield":true
+		"largeshield":true,
+		"milletianshield":true
```
milletianshield shouldn't be a type (just a name of an item), see below

JamEnergy commented on data.js:
```diff
@@ -74,7 +75,8 @@ var g_weaponTypes = {
 		"focus":true,
-		"whip":true
+		"whip":true,
+		"milletianfocus":true
```
same as above

JamEnergy commented on data.js:
```diff
@@ -878,7 +917,56 @@ var g_offhands = [
+	},
+	{
+		"name":"Milletian Focus",
+		"type":"milletianfocus",
```
type is wrong, the shining focus one was right though

JamEnergy commented on data.js:
```diff
@@ -878,7 +917,56 @@ var g_offhands = [
+	},
+	{
+		"name":"Milletian Shield",
+		"type":"milletianshield",
```
type should just be smallshield

JamEnergy reviewed (changes requested):
Did you add new types for militian shield/focus to make the scrolls apply on the appropriate offhand items only? That has the correct effect, but I would for now prefer if you didn't create new item types (accepting the "bug" that you can put the new scrolls on old shields).

Could you make that change and re-submit?

I'll add something like a level or name filter to the scroll to fix the resultant bug of the scroll being applicable to items that it isn't meant to be.

Thanks for taking the effort to understand the code and add to the data, though.

jiglei commented:
I'm sorry if this added categories look ugly, I'll revert it.

I'm not really skilled on this kind of stuff, just wanted to make something functional :)

jiglei committed:
Categories removed

Removed "milletianshield" and "milletianfocus" categories.
The items now belong to "smallshield" and "focus" categories respectively.
This means that I had to add Sealed and Truth ES to be scrollable on all shields and focus. I do not know a way to avoid this.

JamEnergy commented:
Trailing commas are apparently okay in Javascript, although they are illegal JSON. I don't care enough to ask for that to be changed right now though.
